src/style.css

```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  text-transform: uppercase;
+  background-color: #fff;
+  padding: 10px 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo__img {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  background-color: #fff;
+  height: 100%;
+}
+
+.nav__link {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  color: #000;
+  text-decoration: none;
+  padding: 10px 20px;
+
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link::after {
+  content: "";
+  display: block;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+}
+
+.nav__link.is-active::after {
+  background-color: #00acdc;
+  transform: scaleX(1);
+}
+
+.nav__link.is-active {
+  color: #00acdc;
 }
```

Comment:
the link with blue color and line below is an active link. It should have `class="is-active"` and relevant styles.
these styles should be for `is-active`, not for `.nav__link`